include/sustainml/Engine.h

```diff
@@ -24,18 +24,17 @@
 
```

Comment:
We could take the chance to rename this header to `Engine.hpp`

Reply:
I've done it in a [follow up PR](https://github.com/eProsima/SustainML-Framework/pull/18) ([this commit](https://github.com/eProsima/SustainML-Framework/pull/18/commits/44db0c3f0f627d7a5dcad64e42481b21f184f9a1))